src/environments/type.ts

```diff
@@ -0,0 +1,9 @@
+import { Provider } from "@angular/core";
+
+export interface Environment {
+	production: boolean,
+	apiRoot: string,
+	mockProviders: Provider[],
+	locales: string[],
+	apiURL?: string,
```

Comment:
what's the difference between apiRoot and apiURL?

Reply:
it should only be apiRoot now

Reply:
removed `apiURL` from type and the actual value to avoid confusion